src/popup/utils/openUrl.js

```diff
@@ -13,7 +13,7 @@ export default (url, newTab) => {
     }
     case 'cordova':
       document.addEventListener('deviceready', () => {
-        cordova.InAppBrowser.open(url, '_system');
+        cordova.InAppBrowser.open(url, '_blank');
```

Comment:
We were using inappbrowser plugin just to open system browser. Right now it works without the plugin, so I'm proposing to remove it in #943